docs/layers/trips-layer.md

```diff
@@ -20,7 +21,37 @@ npm install @deck.gl/core @deck.gl/layers @deck.gl/geo-layers
 
 ```js
 import {TripsLayer} from '@deck.gl/geo-layers';
-new TripsLayer({});
+
+const App = ({data, viewport}) => {
+
+  /**
+   * Data format:
+   * [
+   *   {
+   *     segments: [
+   *       [-122.269029, 37.80787, 1554772579000], // lng, lat, timestamp
+   *       [-122.269029, 37.80787, 1554772579010],
+   *       ...,
+   *       [-122.271604, 37.803664, 1554772580200]
+   *     ]
+   *   }
+   * ]
+   */
+  const layer = new TripsLayer({
+    id: 'trips-layer',
+    data,
+    // deduct start timestamp from each data point to avoid overflow
+    getPath: d => d.segments.map(p => p[2] = p[2] - 1554772579000),
```

Comment:
This is invalid code.

For absolute clarity, we can change the sample data to this format:

```js
[{
  waypoints: [
    {coordinates: [-122.269029, 37.80787], timestamp: 1554772579000},
    ...
  ]
}]
```

Then

```js
getPath: d => d.waypoints.map(p => [p.coordinates[0], p.coordinates[1], p.timestamp - 1554772579000])
```